features/shop/product/viewing_products/viewing_products_with_multiple_variants_sorted_by_price.feature

```diff
@@ -22,15 +22,15 @@ Feature: Viewing products with multiple variants sorted by price
         And the product "Johnnie Walker" has "Johnnie Walker Black Label" variant priced at "$25.00"
         And the "Johnnie Walker White Label" product variant is disabled
 
-    @ui @api
+    @todo-api @ui
     Scenario: Sorting products by price of their first variant with ascending order
```

Comment:
if I am thinking correctly, these scenarios of sorting products should be achievable 🤔 However, considering the size of the PR, let's already cover this separately